src/components/datepicker/datepicker.jsx

```diff
@@ -0,0 +1,8 @@
+import ReactDatePicker from 'react-datepicker';
+
+import 'react-datepicker/dist/react-datepicker.css';
+import 'react-select/dist/react-select.css';
```

Comment:
I guess `react-select` should stay in `edit-event-form.jsx`.
later @AbBoris will move it to separate component